### PR TITLE
Add service to order copy cards

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-767

Add a reusable service that given a registration, a user, a number of copy cards and a user, generates a copy cards only order

### CODE REVIEW

File: app/models/waste_carriers_engine/order.rb [hunk 1]
```diff
@@ -25,19 +25,12 @@ class Order
     field :manualOrder, as: :manual_order,           type: String
     field :order_item_reference,                     type: String
 
+    # TODO: Move to a service
```
Comment (Cruikshanks) [inline app/models/waste_carriers_engine/order.rb]: If the intention is to move this to an `OrderRenewalService` (or something like that) would it be better to do that refactoring first and then add your new service?

As is I'm just not sure I like the idea that if we never get round to this `TODO` we'll have `Order` create the order with `new_order`, set the order items as if its a renewal, but then `OrderAdditionalCardsService` take that order and simply overwrites the array. Plus we're creating orders in 2 different ways.

Comment (cintamani) [reply app/models/waste_carriers_engine/order.rb]: This service do not use the code in `new_order` so I have left the TODO for the next person touching it. We should get there naturally when we work out the front end flow.

Comment (Cruikshanks) [reply app/models/waste_carriers_engine/order.rb]: 🤦‍♂ Yep, was pointed out to me I was misreading the changes!

**Comment by cintamani** [comment]
I am pretty sure I actually need this as it is now, so I am merging this in

**Pull request merged**